src/main/java/de/slikey/effectlib/Effect.java

```diff
@@ -319,6 +329,15 @@ public void setDynamicTarget(DynamicLocation location) {
     }
 
     protected final boolean validate() {
+        // Check if the origin and target entities are present
+        if (disappearWithOriginEntity && (origin != null && origin.getEntity() == null)) {
+            return false;
```

Comment:
My assumption was that if origin was null, that means that the location wasn't set in the first place, thus it was likely a mistake that they set disappearWithOriginEntity to true